srcpkgs/AppStream/template

```diff
@@ -18,15 +20,24 @@ checksum=c44e1e8f0684a7c3bfce4c366148956ac29fa0b7c8298ef9952faf79bbb34fef
 conf_files="/etc/appstream.conf"
 triggers="appstream-cache"
 
-nocross="https://api.travis-ci.org/v3/job/592923595/log.txt"
+build_options="gir vala"
+build_options_default="gir vala"
 
 AppStream-devel_package() {
 	depends="AppStream-${version}_${revision} AppStream-qt-${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
+		vmove usr/lib/cmake
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		if [ "$build_option_gir" ]; then
+			vmove usr/lib/girepository-1.0
```

Comment:
`usr/lib/girepository-1.0` should be installed in main/libraries package.

Reply:
Done.

Reply:
Thanks